test/stripe/stripe_client_test.rb

```diff
@@ -1145,6 +1192,193 @@ class StripeClientTest < Test::Unit::TestCase
       end
     end
 
+    context "API resource instance methods" do
```

Comment:
I'm not 100% sold on this testing approach. It feels too easy for regressions to find there way in, but maybe I've stared at it for too long.